types/parse/index.d.ts

```diff
@@ -19,6 +19,64 @@
 
 /// <reference types="node" />
 
+declare enum ErrorCode {
+    OTHER_CAUSE = -1,
```

Comment:
If you want to be able to import these from a consuming file, they need to be within the `Parse` namespace.

Reply:
@uniqueiniquity they are exported as static variables on the `Parse.Error` class in the changes below, [starting at line 968](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/38472/files#diff-aec51b2d87715b12fe03e0f09983ea8fR968). I followed the advice from the previous abandoned PR in [this comment](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/31248#pullrequestreview-183468400). The reasoning behind not declaring the enum inside the class is that the entity `Parse.ErrorCode` doesn't exist on Parse object.

The import appears to work when I test this file in my local `node_modules` folder. Is there something more that I'm not understanding? Thanks for your help!

Reply:
Makes sense to me, I just wanted to check. Thanks for the explanation!